CMakeLists.txt

```diff
@@ -775,8 +775,6 @@ MafReadDir(srtcore filelist.maf
 	PRIVATE_HEADERS HEADERS_srt_private
 )
 
-message(STATUS "SRT Sources: ${SOURCES_srt}")
-
```

Comment:
Unrelated :) OTOH I don't think this is necessary, although it's useful sometimes when you change the source list and try to debug the build scripts.